.github/workflows/build-unitycloud.yml

```diff
@@ -12,6 +12,8 @@ on:
   push:
     branches:
       - dev
+    branches-ignore:
+      - main
```

Comment:
I think this means when a push happens to main, it won't try and build unity-cloud. But the action build-release-main should run still (as it will use workflow_dispatch) ?

Reply:
hmm, I still think that the first should be enough. And we can probably remove workflow_dispatch from the Unity Cloud Build Release, as that should only work when we merge to main.